texture_converter.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf8 -*-
+import png
+from struct import unpack
+import sys
+import os
+from PyQt4 import QtGui, QtCore
+import tempfile
+
+class Converter(object):
+    def convert_bump(self, pixels, width, height, nb_components):
+        outp = []
+        for y in range(1, height - 1):
+            w = []
+            for x in range(1, (width * nb_components) -1, nb_components):
+                norm_val = QtGui.QVector3D(pixels[y][x+1] - pixels[y][x-1], pixels[y+1][x] - pixels[y-1][x], 255)
+                norm_val.normalize()
+                norm_val = norm_val.__mul__(QtGui.QVector3D(127, 127, 127))
```

Comment:
it means it can't work from command line right ? you need to have qt4 installed. If it's the only deps maybe we can add some math func in the code instead of using qt4